README/ReleaseNotes/v630/index.md

```diff
@@ -247,6 +247,8 @@ Some of these classes are now removed from the public interface:
 
 ## 2D Graphics Libraries
 
+- Implement `TAxis::ChangeValueLabel` to set custom label defined by axis value. It is more
+  robust method, which continue working when axis is zoomed.
```

Comment:
It is not "more robust". it is a different way. We still need to be able to change the label according to its position.
Conn you change this comment ?

Reply:
I adjust release comments